index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header id="header">
+    <div class="header-container">
+      <div class="logo">
+        <a href="https://github.com/OLIPLICHE/portfolio/pull/1"> My Logo</a>
+      </div>
+     <nav class="navbar">
+      <ul class="d-navbar">
+        <li><a class="a-navbar" href="#Portfolio">Portfolio</a></li>
+        <li><a class="a-navbar" href="#About">About</a></li>
+        <li><a class="a-navbar" href="#Contact">Contact</a></li>
+      </ul>
+      <div class="humberger" >
+        <i class="fa fa-bars"></i>
+      </div>
+    </nav>
+   </div>
+  </header>
+  <main id="main-container">
+    <section class="headline">
+      <div class="headline-main-container">
+        <h1 class="welcome">I am Paka <br> Glad to see you!</h1>
```

Comment:
Let's make this h1 look like the original one and also use the property line-height in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design

Example:
![image](https://user-images.githubusercontent.com/31897434/125872540-85e6fd93-0024-4698-b32d-1d33e2290cb0.png)

Reply:
This issue was well resolved ✔️